src/ui/profile_preferences.ui

```diff
@@ -3380,6 +3409,15 @@
                     </color>
                    </brush>
                   </colorrole>
+                  <colorrole role="PlaceholderText">
```

Comment:
Aaargh! - Another bug with the Qt Designer plugin / utility - is that it assumes that the `.ui` files that it makes are only going to be used by the current Qt version (specifically the `QUILoader` class) of the libraries and it will happily put in details that are un-parsable and will break on previous versions of Qt - and the `PlaceholderText` element of a `QPalette` is one of them - it was introduced in, I think Qt 5.12 but our code must be compilable going back to Qt **5.11** versions - therefore you will need to manually (with the plain text editor) going to have to excise all the `<colorrole role="PlaceholderText">`...`</colorrole>` elements in the file... {though the previous comment's hint I gave about the element folding will help you to do this. :smiling_imp: }

Reply:
That is why the `.7` job of the Travis Linux CI build is failing - it specifically runs on the oldest Qt version we support to test for this sort of thing. :grin:

Reply:
Fixing. Yes I do know how to use an XML editor. ;-)